specification/datadog/resource-manager/Microsoft.Datadog/stable/2023-01-01/datadog.json

```diff
@@ -2004,8 +1970,7 @@
       "x-ms-enum": {
         "modelAsString": true,
         "name": "ProvisioningState"
-      },
-      "readOnly": true
```

Comment:
>readOnly": true [](http://example.com/codeflow?start=7&length=15)

why did you removed the readonly from ProvisioningState?

Reply:
Same has been reverted in private repo and we want to be in sync with it.
https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/datadog/resource-manager/Microsoft.Datadog/stable/2023-01-01/swagger.json

Besides this value can not be set by user, adding it here will make us out of sync from private repo and can led to issue. 
We will add this back in new version to avoid breaking change here.